WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsUsageTracksEventEmitter.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+/// Note: If we ever change the algorithm in the future, we should probably consider renaming
+/// the Tracks event to avoid incorrect comparisons with old events.
+final class StoreStatsUsageTracksEventEmitter {
+
+    /// TODO Replace with proper injection
+    static let shared = StoreStatsUsageTracksEventEmitter()
+
+    private let analytics: Analytics
+
+    /// The minimum amount of time (seconds) that the merchant have interacted with the
+    /// Analytics UI before an event is triggered.
+    private let minimumInteractionTime: TimeInterval = 10
+
+    /// The minimum number of Analytics UI interactions before an event is triggered.
+    ///
+    /// The interactions captured are:
+    ///
+    /// - Scrolling
+    /// - Pull-to-refresh
+    /// - Tapping on the bars in the chart
+    /// - Changing the tab
+    /// - Navigating to the My Store tab
```

Comment:
How about tapping on a product in the top performers section?

Reply:
Added in ea5fa60ea597affb0dc0f4fe02cafaeab1debac2 (draft PR #5820)